lunes_cms/api/v1/serializers/alternative_word_serializer.py

```diff
@@ -9,10 +9,12 @@ class AlternativeWordSerializer(serializers.ModelSerializer):
     `serializers.ModelSerializer`.
     """
 
+    article = serializers.IntegerField(source="singular_article")
+
     class Meta:
         """
         Define model and the corresponding fields
         """
 
         model = AlternativeWord
-        fields = ("alt_word", "singular_article")
+        fields = ("alt_word", "article")
```

Comment:
As https://github.com/digitalfabrik/lunes-app/pull/1024/files is already merged, we should support both attributes.

Reply:
No, it has not yet been merged :see_no_evil: I have no idea which PR I accidentally visited to see that. Then we can merge your PR IMHO.